data/data-pipeline/data_pipeline/etl/sources/national_risk_index/etl.py

```diff
@@ -153,6 +153,19 @@ def transform(self) -> None:
             lower=self.AGRIVALUE_LOWER_BOUND
         )
 
+        ## Check that this clip worked -- that the only place the value has changed is when the clip took effect
```

Comment:
This isn't the nicest -- but we don't port out of the ETL a lot of these columns, so I check in place. Is it too sloppy?

Reply:
Honestly the only thing I'd change is the error message in the assertion at 167 --- like, can you be any more descriptive about how the clip didn't work? Otherwise, this fits the pattern we discussed pretty well of adding assertions within the code to programmatically enforce our assumptions.

Reply:
👍 thank you so much! updated